vortex-serde/src/layouts/mod.rs

```diff
@@ -5,7 +5,14 @@ mod pruning;
 #[cfg(test)]
 mod tests;
 
+pub const VERSION: u16 = 1;
 pub const MAGIC_BYTES: [u8; 4] = *b"VRX1";
```

Comment:
kind of trivial, but since we're breaking all existing files *anyway*, why not make this "VRTX" instead of "VRX1"?